test/unit/DCAStrategies/dca-strategies-positions-handler.spec.ts

```diff
@@ -82,6 +135,34 @@ contract('DCAStrategiesPositionsHandler', () => {
       then('transferFrom() is called correctly', async () => {
         expect(tokenA.transferFrom).to.have.been.calledOnceWith(user.address, DCAStrategiesPositionsHandlerMock.address, toDeposit);
       });
+      then('_approveHub() is called correctly', async () => {
+        expect(tokenA.approve).to.have.been.calledOnceWith(hub.address, constants.MAX_UINT_256);
+        let approveHubCalls = await DCAStrategiesPositionsHandlerMock.getApproveHubCalls();
+        expect(approveHubCalls[0].token).to.be.equal(tokenA.address);
+        expect(approveHubCalls[0].hub).to.be.equal(hub.address);
+        expect(approveHubCalls[0].amount).to.be.equal(toDeposit);
+      });
+      then('deposit() in hub is called correctly', async () => {
+        expect(hub['deposit(address,address,uint256,uint32,uint32,address,(address,uint8[])[])']).to.have.been.calledTwice;
+        expect(hub['deposit(address,address,uint256,uint32,uint32,address,(address,uint8[])[])'].atCall(0)).to.have.been.calledWith(
+          tokenA.address,
+          tokenB.address,
+          toDeposit.div(2),
+          amountOfSwaps,
+          swapInterval,
+          DCAStrategiesPositionsHandlerMock.address,
+          []
+        );
+        expect(hub['deposit(address,address,uint256,uint32,uint32,address,(address,uint8[])[])'].atCall(1)).to.have.been.calledWith(
+          tokenA.address,
+          tokenC.address,
+          toDeposit.div(2),
```

Comment:
Could you make `toDeposit` an odd number, so that we need to change `toDeposit.div(2)` for `toDeposit.sub(toDeposit.div(2))`?